src/components/style/styling.css

```diff
@@ -0,0 +1,22 @@
+.adminPageHeader {
```

Comment:
I would prefer to make use of the `sx` property for one-off styling or creating [reusable components](https://mui.com/material-ui/customization/theming/#accessing-the-theme-in-a-component) . MUI reusable components and `sx` is much easier to manage and makes conditional styling easier.

Reply:
removed the CSS file and folder. Added the style to the appropriate tags